Documentation/Input/Pointers.md

```diff
@@ -1,39 +1,235 @@
+> MRTK provides a set of pointer prefabs in *Assets/MixedRealityToolkit.SDK/Features/UX/Prefabs/Pointers*. A new custom prefab can be built as long as it contains one of the pointer scripts in *Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers* or any other script implementing [`IMixedRealityPointer`](xref:Microsoft.MixedReality.Toolkit.Input.IMixedRealityPointer).
+
+### Default pointer classes
+
+The following classes are the out-of-box MRTK pointers available and defined in the default *MRTK Pointer Profile* outlined above. Each pointer prefab provided under *Assets/MixedRealityToolkit.SDK/Features/UX/Prefabs/Pointers* contains one of these pointer components attached.
+
+![MRTK Default Pointers](../Images/Input/Pointers/MRTK_Pointers.png)
+
+#### Far pointers
+
+##### [`LinePointer`](xref:Microsoft.MixedReality.Toolkit.Input.LinePointer)
+
+ A base pointer class, draws lines from the source of the input (i.e. the controller) in the pointer direction. Generally, children classes such  the [`ShellHandRayPointer`](xref:Microsoft.MixedReality.Toolkit.Input.ShellHandRayPointer) and the teleport pointers are instantiated and utilized (which also draw lines to indicate where teleportation will end up at) instead of this class which primarily provides common functionality.
+
+For motion controllers like in Oculus, Vive, Windows Mixed Reality, the rotation will match the rotation of the controller. For other controllers like HoloLens 2 articulated hands, the rotation matches the system-provided pointing pose of the hand.
+
+<img src="../../Documentation/Images/Pointers/MRTK_Pointers_Line.png" width="400">
+
+##### [`ShellHandRayPointer`](xref:Microsoft.MixedReality.Toolkit.Input.ShellHandRayPointer)
+
+The implementation type, which extends from [`LinePointer`](xref:Microsoft.MixedReality.Toolkit.Input.MousePointer), used as the default for the *MRTK Pointer Profile*. The *DefaultControllerPointer* prefab implements the [`ShellHandRayPointer`](xref:Microsoft.MixedReality.Toolkit.Input.ShellHandRayPointer) class.
+
+##### [`GGVPointer`](xref:Microsoft.MixedReality.Toolkit.Input.GGVPointer)
+
+Also known as the Gaze/Gesture/Voice pointer, the GGVPointer powers HoloLens 1-style look and tap interactions, primarily via Gaze and Air Tap or Gaze and voice Select interaction. The GGV pointer's position and direction is driven by the head's position and rotation.
+
+##### [`TouchPointer`](xref:Microsoft.MixedReality.Toolkit.Input.TouchPointer)
+
+The TouchPointer is responsible for working with Unity Touch input (i.e. touchscreen). These are 'far interactions' because the act of touching the screen will cast a ray from the camera to a potentially far location in the scene.
+
+##### [`MousePointer`](xref:Microsoft.MixedReality.Toolkit.Input.MousePointer)
+
+The MousePointer powers a screen to world raycast for far interactions, but for mouse instead of touch.
+
+<img src="../../Documentation/Images/Pointers/MRTK_MousePointer.png" width="400">
+
```

Comment:
` [!NOTE] Mouse support is not available by default in MRTK and can be enabled by adding a new *Input Data Provider* of type @Microsoft.MixedReality.Toolkit.Input.UnityInput.MouseDeviceManager to the MRTK input profile.`